storybook/src/components/Dialog/Dialog.docs.mdx

```diff
@@ -14,6 +14,18 @@ import README from "../../../../packages/css/src/components/dialog/README.md?raw
 
 ## Examples
 
+### Form dialog
+
+The `<form method="dialog">` element is a practical way to manage dialogs.
+It simplifies the process by automatically handling the closure and data submission
+when a user interacts with a submit or cancel button. For more complex scenarios,
+developers can opt for custom handling by adding the form as a child element.
+Additionally, creating sticky buttons in a dialog's footer is achievable by placing
+a submit button outside the `<form>` and associating it with the form's ID using the
+`form` attribute.
```

Comment:
```suggestion
### Form in a Dialog

Set `method="dialog"` when using a form in Dialog.
This closes the Dialog when submitting the form.
Pass the submit Button to the `footer` prop,
and link it to the form by passing its `id` to the Buttons `form` attribute.
```

Reply:
Do we have to capitalise every mention of a component?

Reply:
[Yup](https://github.com/Amsterdam/design-system/blob/develop/documentation/storybook.md#structure:~:text=Component%20names%20are%20in%20title%20case%20%E2%80%93%20starting%20each%20word%20with%20a%20capital%20letter%20%E2%80%93%20to%20make%20people%20recognise%20them%20as%20such.)